packages/codemods/transforms/v4/dropdown-list-item-interactive/README.md

```diff
@@ -0,0 +1,35 @@
+# v4/icon
+
+## Usage
+
+To run this codemod in your project using `npx`, you would run the following:
+
+```bash
+npx @hashicorp/design-system-codemods v4/dropdown-list-item-interactive path/to/some/glob/**/*.hbs
+```
+
+## Local usage
+
+To run this codemod in this repository (even before publishing it), you would run the following from the root directory of this repository:
+
+```bash
+node ./packages/codemods/bin/cli.js v4/dropdown-list-item-interactive path/to/some/glob/**/*.hbs
+```
+
+## Input
+
+```hbs
+<Hds::Dropdown as |dd|>
+  <dd.Title @text='Title text' />
+  <dd.Interactive @href='#' @text='Edit' />
```

Comment:
The HDS team normally uses double quotes within HTML and components to enclose argument values. (We use single quotes within JS/TS code however.)

Could you please open another small PR to update the examples? (No need to create a new ticket.)